packages/database/src/migrations/20220512232924-CreateDhisInstanceTable-modifies-schema.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    CREATE TABLE dhis_instance (
+      id TEXT PRIMARY KEY,
+      code TEXT NOT NULL UNIQUE,
+      readonly BOOLEAN NOT NULL,
+      config JSONB NOT NULL
```

Comment:
I'm perfectly happy with raw SQL here, but incase you weren't aware (I wasn't until recently) there's a `db.createTable()` function 👍